### ISSUE TITLE
Docs : Revamp of the doc

### ISSUE THREAD

**Issue by xexax** [arita37/mlmodels#331]
[https://docs.google.com/document/d/e/2PACX-1vTDq5YVexX5Cz4IrP3Hyc66Blc9ReLWB-R4Djj6DKYEFw5wNZNLvy8aeDICh5rDx_YpcW1ZSVqth98a/pub](url)

**Comment by xexax** [comment]
README.MD
Hi, I wanted to point what confused me at first glance and tried to make it seen from a user point of view. In general it was confusing which roles are in game users/contributors/devs

For beginner intermediate users,The end to end cycle of the mlmodels repo is not clear, an About section would clarify what is it / what aims for. 

Readme starts with mlmodels:Model ZOO, does it mean it is related to https://modelzoo.co/ or it is the same logic a ZOO for the mlmodels

In the main characteristics there is a shout(We are looking for contributors (!!)), what level of knowledge is required expert/non expert, which domains are required

There are logos shared but actually repo is much wider, does the 3 below main ones?

Benefits is confusing, maybe better to change it as “ benefits of mlmodels project”


After clicking usages guide 
I am directed to mlmodels/mlmodels/example/README_usage.md
 and before starting all the setup,  a simple summary of what all those actions will bring and why i should do all these installs would help me if I should proceed or not.
	Inside the guide which one I should choose, between pre installed setup or colab. 






Model list section : 
7.1.Does this section includes all the models that are already covered in mlmodels zoo or a contributor should take one of the links and start working on it. 
7.2.To make the first read me non confusing can they all be moved under README_model_list.md?






Sections without info, in an example Auto-KEras :2020, should a note be shared in development?


info:There is no more resources in the link, maybe a line could be added “will be updated after 10/06/2020 by xyz(contributor)”

After the resources there is additional Installation guide but previously there was already a guide.

There are examples but we already had examples readme are they same ?

**Comment by arita37** [comment]
Can we move docs here :
   docs/dev-doc/  : For developper
   docs/user-doc/ : For user only

   And in github/isssues for : 
      Tutorial, Examples (easy to update by all)

**Comment by xexax** [comment]
> Can we move docs here :
> docs/dev-doc/ : For developper
> docs/user-doc/ : For user only
> 
> And in github/isssues for :
> Tutorial, Examples (easy to update by all)
> 
> https://github.com/arita37/mlmodels/labels/Colab

docs/dev-doc/ : For developer ->subfolder exists - should I move docs there?
docs/user-doc/ : For user only -> should I create a new subfolder and move user folders here?

**Comment by arita37** [comment]
Not move, make a copy into :
https://github.com/arita37/mlmodels/tree/dev/docs/DEV_docs


Want to keep some readme on 1st page

**Comment by xexax** [comment]
> Can we move docs here :
> docs/dev-doc/ : For developper
> docs/user-doc/ : For user only
> 
> And in github/isssues for :
> Tutorial, Examples (easy to update by all)
> 
> https://github.com/arita37/mlmodels/labels/Colab

> Not move, make a copy into : -> SS:okay I will copy
> https://github.com/arita37/mlmodels/tree/dev/docs/DEV_docs
> 
> Want to keep some readme on 1st page
> 
> What do you think of github/issues system ? -->  SS: It is great but can I make a tracking/overview file for open issues? (not sure if there is already an easy way to track all open issues). If you think unnecessary and there is a better way to see overall status using only github I would be more than happy to learn.
> 
> Easy to extend with no effort ? -> yes, maybe too easy? i.e. maybe there is already open issue and someone can open again a similar issue?

**Comment by arita37** [comment]
We need moderator.
Goal is to allow contribution of notebooks very easily....

**Comment by arita37** [comment]
Ok for tracking but nothing manual....
Everything should be automated by github Actions 
Or github API.

Lack of people doing the mainteance.....

**Comment by xexax** [comment]
Hi, Tutorial : New contributors #307 

Created a copy of the tutorial to the directory under docs/dev-doc/ , is it aligned with what you asked?

If it is I will continue with all user-docs and dev_docs in a similar way,
1.enter to issues 
2.filter labels dev docs create copy of each documentation under issues in markdown format 
under docs/dev-doc/ : For developper
filter labels user docs create copy of each documentation under issues in markdown format 
underdocs/user-doc/ : For user only


> Not move, make a copy into :
> https://github.com/arita37/mlmodels/tree/dev/docs/DEV_docs

**Comment by arita37** [comment]
Best is :
   Create a separate repo for developper, where we put docs/issues there.
   Keep this repo for users only (no code devs).

**Comment by arita37** [comment]
Instead of filling with contents, can you work on the layout (filename, content) 
in that folder ?

https://github.com/arita37/mlmodels/tree/dev/docs/DEV_docs


Because content will either in 
   .md files  or in Github/Issues Post

https://github.com/arita37/mlmodels/issues?q=is%3Aissue+is%3Aopen+label%3Adev-documentation

If the Github/Issues is here, maybe there is no need of duplicate,
we can put hyperlinks or update the Github/Issues

**Comment by arita37** [comment]
Can you write down the new doc plan / structure ?

**Comment by arita37** [comment]
Developer Documentation :

https://github.com/arita37/mlmodels/issues?q=is%3Aopen+is%3Aissue+label%3Adocumentation

https://github.com/arita37/mlmodels/blob/dev/README.md

End User Documentation :
https://github.com/arita37/mlmodels/issues?q=is%3Aopen+is%3Aissue+label%3Adocumentation

Using Issues as Documentation

https://github.com/arita37/mlmodels/issues?q=is%3Aopen+is%3Aissue+label%3Adev-documentation

using markdown
